_includes/one-click-payments.md

```diff
@@ -85,6 +87,33 @@ GET {{ authorization_url }} HTTP/1.1
 Authorization: Bearer <AccessToken>
 ```
 
+{% else %}
+
+When the initial purchase is successful, a `paymentToken` is linked to
+the payment. You can return the value by sending a `GET` request towards the
+payment resource with the `payerReference` included.
+
+{:.code-view-header}
+**Request Towards The Payment Resource**
+
+```http
+GET /psp/paymentorders/payerownedtokens/{payerReference} HTTP/1.1
+Authorization: Bearer <AccessToken>
+```
+
+You can also send a GET request towards the `paymentOrderId` and find the
+paymentToken in the `paid` resource.
+
+{:.code-view-header}
+**Request Towards The Payment Resource**
```

Comment:
Before performing the request towards the `paid` resource, we should show how the URL to it can be found within a payment order.

```suggestion
**Request Towards The Paid Resource**
```